src/model/liveposition.js

```diff
@@ -34,8 +34,7 @@ export default class LivePosition extends Position {
 	 * @see module:engine/model/position~Position
 	 * @param {module:engine/model/rootelement~RootElement} root
 	 * @param {Array.<Number>} path
-	 * @param {module:engine/model/position~PositionStickiness} [stickiness] Position stickiness. Defaults to `'toNone'`.
-	 * See {@link module:engine/model/liveposition~LivePosition#stickiness}.
+	 * @param {module:engine/model/position~PositionStickiness} [stickiness]
```

Comment:
[stickiness='toNone']? Any docs?

Reply:
Stickiness is described in typedef. I could go with something like `See {@link ... }` but this is a bit stupid cause in generated docs you will have a link to the typedef next to the parameter.